src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -20,7 +20,7 @@ import org.joda.time.DateTime
 object ErrorAggregator {
 
   private val allowedDocTypes = List("main", "crash")
-  private val outputPrefix = "error_aggregates/v1"
+  private val outputPrefix = "error_aggregates/v2"
```

Comment:
I think it does require a version bump. From my understanding if we change the dimensions schema the aggregations in flux are not valid anymore, so we cannot just restart the job.